pkg/reconciler/v1alpha1/service/service.go

```diff
@@ -244,13 +244,16 @@ func (c *Reconciler) reconcile(ctx context.Context, service *v1alpha1.Service) e
 				want[idx].ConfigurationName = ""
 			}
 		}
-		ignoreURLs := cmp.FilterPath(
+		ignoreFields := cmp.FilterPath(
 			func(p cmp.Path) bool {
-				return p.String() == "URL"
+				return p.String() == "TrafficTarget.URL" ||
```

Comment:
So I know I wrote this originally, but IgnoreFields() from cmpopts might be easier to read.

```
cmpopts.IgnoreFields(TrafficTarget{}, "TrafficTarget.Url", ...)
```

Reply:
I bow to your mighty `go-cmp` knowledge.